doc/release/1.7.0-notes.rst

```diff
@@ -24,53 +24,290 @@ For running on PyPy, PyPy3 6.0+ is required.
+  distribution, respectively.
+
+The ``cosine`` distribution has added ufuncs for ``ppf``, ``cdf``, ``sf``, and
+``isf`` methods including numerical precision improvements at the edges of the
+support of the distribution.
+
+The function `scipy.stats.mannwhitenyu` has been rewritten. There are two 
+changes to default behavior:
+
+- For years, use of the default ``alternative=None`` was deprecated; explicit 
+  ``alternative`` specification was required. Use of the new default value of
+  ``alternative``, "two-sided", is now permitted.
+- Previously, all p-values were based on an asymptotic approximation. Now, for
+  small samples without ties, the p-values returned are exact by default.
+
+The new ``axis`` parameter of `scipy.stats.mannwhitenyu` allows vectorized
+computation of the test statistic and p-values along a given axis of the
+provided samples. The new parameter ``method`` enables control over the method
+used for p-value calculation ("auto", "exact", or "asymptotic"). Note that the
+exact calculation is based on the null distribution of the statistic derived
+without ties.
+
+The new function `scipy.stats.binomtest` replaces `scipy.stats.binom_test`. The
+new function returns an object that allows confidence intervals for the
+estimated proportion to be computed. Also, performance was improved from O(n)
+to O(log(n)) by using binary search.
+
+The new function `scipy.stats.contingency.crosstab` computes a contingency
+table (i.e. a table of counts of unique entries) for the given data.
+
+The new function `scipy.stats.contingency.association` computes several
+measures of association for a contingency table: Pearsons contingency
+coefficient, Cramer's V, and Tschuprow's T.
+
+The Alexander-Govern test is implemented in the new function
+`scipy.stats.alexandergovern`.
+
+The new function `scipy.stats.barnard_exact` performs Barnard exact test for
+2x2 contingency tables.
+
+The new function `scipy.stats.differential_entropy` calculates the differential
+entropy with five ``method`` values: ``{'vasicek', 'van es', 'ebrahimi',
+'correa', 'auto'}``.
+
+The new function `scipy.stats.page_trend_test` performs Page's test for ordered
+alternatives.
+
+The new function `scipy.stats.somersd` computes Somers' D, a measure of ordinal
+association between two variables.
+
+The parameter ``nan_policy`` was added to `scipy.stats.zmap` to provide options
+for handling the occurrence of ``nan`` in the input data.
+
+The parameter ``ddof`` was added to `scipy.stats.variation` and
+`scipy.stats.mstats.variation`.
+
+The parameter ``weights`` was added to `scipy.stats.gmean`.
+
+An option to fit the distribution to data by the method of moments has been
+added to the ``fit`` method of the univariate continuous distributions.
+
+We gratefully acknowledge the Chan-Zuckerberg Initiative Essential Open Source
+Software for Science program for supporting many of these improvements to
+`scipy.stats`.
+
+A new `scipy.stats.qmc` module for quasi-Monte Carlo
```

Comment:
I suggest making this a subsection of the `scipy.stats improvements` section:
```
A new `scipy.stats.qmc` module   # underline with backticks
```

Reply:
I'm also making sections for Hypothesis Tests, Distributions, and Sample Statistics

Reply:
It looks like the subsections used the wrong underlines so I just changed them to regular sections, but Ralf's comment suggests it may be "\`" should have been used instead of "^"?

```
Hypothesis Tests
^^^^^^^^^^^^^^^^
release/1.7.0-notes.rst:184: WARNING: Title level inconsistent:

Sample statistics
^^^^^^^^^^^^^^^^^
release/1.7.0-notes.rst:214: WARNING: Title level inconsistent:

Statistical Distributions
^^^^^^^^^^^^^^^^^^^^^^^^^
release/1.7.0-notes.rst:247: WARNING: Title level inconsistent:

Other
^^^^^
release/1.7.0-notes.rst:255: WARNING: Title level inconsistent:

New `scipy.stats.qmc` module
^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```